test/utility-test.js

```diff
@@ -107,108 +107,6 @@ describe('Database Util', function() {
       });
     });
   });
-  describe('#getRowById()', function() {
```

Comment:
getRowById, getRowsByIds and getLatestRow use the same function call under the hood and we have getLatestRow 'tested' already :)